src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+While these events are often focused on producing or proposing new software or hardware as a means of addressing a social problem or creating a new product, this is not stricly necessary; hackathons have been held to produce
+<a href="https://www.policyhack.com.au/" target="_blank" class="link-external">government policy</a>, or ideas to address <a href="http://foodinnovationprogram.org/hacking-hunger/" target="_blank" class="link-external">food security</a>.
+
+In learning settings hackathons might be held as part of course curriculum or as an extracurricular activity for interested students. Academic institutions might seek to partner with community groups for hackathons to encourage students and faculty to form connections outside the institution.
+
+### Hacking
+
+We use the term "hacking" expansively here to refer to ideas of in-depth learning by doing, do-it-yourself and <a href="https://en.wikipedia.org/wiki/Hacker_culture" target="_blank" class="link-external">"playful cleverness"</a> that we can also find present in the term "making". We feel (though others might disagree) that the concepts have enough overlap to discuss them together.
+
+We also consider the hacking approach to be present in one-to-one customizations, adaptations and transformations of technology that allow participation in activities or the use of technology by more diverse groups, including people with disabilities. This approach is found in groups like <a href="http://www.tetrasociety.org/" target="_blank" class="link-external">The Tetra Society of North America</a>, who create customized assistive devices for individuals.
+
+In learning settings, hacking might include classroom projects involving prototyping a product or system, or customizing devices to suit someone's unique needs.
+
+## Inclusively Designing Hackathons and Hacking
+
+It's been said about hackathons that the connections and ideas that get made between people attending are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
+
+In thinking about inclusive design for hackathons, we must ask ourselves some of the same questions we ask in other contexts:
+
+### Asking Who Is Excluded
+
+We must consider <a href="WhatIsTheApproach.md#whoIsExcluded">who might be excluded?</a>, and how to redesign our approach to running, promoting and recruiting for our events.
+
+This is especially critical for hackathons trying to tackle social challenges faced by identifiable groups such as people with disabilities, under the principle of <a href="https://en.wikipedia.org/wiki/Nothing_About_Us_Without_Us" target="_blank" class="link-external">*Nothing About Us Without Us*</a>. Any extra work involved in broadening the reach of the event beyond the "natural" demographic will be balanced out by better outcomes and learning.
```

Comment:
It's a good point - current phrasing is definitely problematic. I've rewritten the sentence a few times trying to get it to sound right and am still not happy with it. Writing thoughts out here (less for feedback then to get them recorded somewhere outside my own head)...

- I think that "nothing about us without us" is an important principle for all kinds of (for lack of better phrasing) technocratic engagement with social problems that hackathons often raise. I'm thinking not just of disabilities, but of poverty, food security, racism, gender discrimination... How do we get across the idea that those most directly affected by social challenges have their own kind of expertise, their own knowledge of great value, that technical solutions to social challenges disconnected from the knowledge and experience of those who experience the social challenges will at best be of limited effectiveness and may even be actively harmful?

- Focusing on topics that are relevant, important and engaging - how do we start to engage the individuals who can share this knowledge with us? Most of us who have the capacity to run or participate in a hackathon already have a considerable level of privilege.